packages/mobile/src/hooks/chat/useChatMessage.ts

```diff
@@ -450,6 +464,18 @@ export const useChatMessage = (props?: {
         return false;
       },
     }),
+    getExpanded: useQuery({
```

Comment:
Perhaps move this part to a separate hook? It uses different props to fetch so only makes the hook more complex than it actually is